textdb/textdb-dataflow/pom.xml

```diff
@@ -59,13 +59,16 @@
             <version>3.6.0</version>
             <classifier>models</classifier>
         </dependency>
-
-
         <dependency>
             <groupId>org.mockito</groupId>
             <artifactId>mockito-all</artifactId>
             <version>1.9.5</version>
         </dependency>
+        <dependency>
+            <groupId>com.google.code.gson</groupId>
```

Comment:
we already have org.json as a dependency for processing JSON. Is there anything that `gson` can do and `org.json` can't, or `gson` can do it significantly easier? Adding more dependencies makes the compile time slower and slower